docs/user_guide/perf_analyzer.md

```diff
@@ -196,6 +196,38 @@ components of the latency. Follow these steps:
 - Select "Upload" and upload the file
 - Select "Replace data at selected cell" and then select the "Import data" button
 
+### Server-side metrics
+
+Perf Analyzer can optionally collect and output server-side metrics, such as GPU
+utilization and GPU power usage. To enable this collection/output, use the
+`--verbose-csv` (with the `-f <filename>` option) and `--collect-metrics`
+options. Perf Analyzer accesses the
+[metrics endpoint](metrics.md) of the Triton server throughout profiling and
+reads several metrics from it to output to a csv file. Each metric column in the
+output csv will have this format:
+
+`<gpu-uuid-0>:<metric-value>;<gpu-uuid-1>:<metric-value>;...;`
+
+The server metrics are collected every N milliseconds (default `1000`) as set by
+the Perf Analyzer `--metrics-interval` option. All metric collections are then
+aggregated (averaged, maximum, etc) across the profiling (during stable
```

Comment:
Be specific about which metrics are averaged/maxed and why.

Reply:
Added explanations 👍